doc/dev/cephfs-mirroring.rst

```diff
@@ -165,7 +165,7 @@ To stop a mirroring directory snapshots use::
   $ ceph fs snapshot mirror remove <fs_name> <path>
 
 Only absolute directory paths are allowed. Also, paths are normalized by the mirroring
-module, therefore, `/a/b/../b` is equivalent to `/a/b`.
+module, therefore, `/a/b/../b` is equivalent to `/a/b`::
```

Comment:
This should be handled with a ``.. prompt:: bash $`` directive, but I'll merge and backport this and then alter it in a future PR.

Reply:
There are lot of places to change then. @zdover23 Please do it in another PR that fixes all places.

Reply:
This file is 409 lines long. I'll add prompts to it, but I'll do it in four separate pull requests so that they can each be reviewed in a reasonably short time.

Reply:
Here is the first of the four PRs that improve the formatting (and grammar) of this file:

https://github.com/ceph/ceph/pull/63256

Reply:
docs refinement is necessarily an incremental process, and will be much more difficult in two months.